helpme/api/views.py

```diff
@@ -20,7 +20,7 @@ class CreateTicketAPIView(LoginRequiredMixin, TicketMetaMixin, CreateAPIView):
     queryset = Ticket.objects.all()
 
     # make separate function so it can be overriden with a different template
-    def send_email(self, serializer, instance):
+    def send_email(self, serializer, instance, support_email):
```

Comment:
Not a bog one but would it be a little more clear if "support_email" was called "reply_to_email"